Package/gulpfile.js

```diff
@@ -112,7 +112,7 @@ const copyAndroidFiles = async () => {
   });
 
   await new Promise(resolve => {
-          gulp.src('../Apps/Playground/node_modules/@babylonjs/react-native/android/build/intermediates/library_and_local_jars_jni/release/**/*')
+          gulp.src('../Apps/Playground/node_modules/@babylonjs/react-native/android/build/intermediates/library_and_local_jars_jni/release/jni/**/*')
```

Comment:
@ryantrem let us know if this jni subfolder doesn't get generated on mac. Not sure if its a pc vs mac difference.

Reply:
This worked correctly on Mac.